### ISSUE TITLE
OnSeekBarChangeListener not working

### ISSUE THREAD

**Issue by lobothijau** [Mohammed-Alaa/RangeSeekBar#3]
I want to listen to value changes by implementing:

```java
seekBar.setOnSeekBarChangeListener(object : SeekBar.OnSeekBarChangeListener {
                override fun onProgressChanged(
                    seekBar: SeekBar?,
                    progress: Int,
                    fromUser: Boolean
                ) {
                    emitter.onNext(progress);
                }

                override fun onStartTrackingTouch(seekBar: SeekBar?) {

                }

                override fun onStopTrackingTouch(seekBar: SeekBar?) {

                }

            })
```

But, when the app run I can't slide and after removing the code I can slide again.

**Comment by MohammedAlaaMorsi** [comment]
@lobothijau I had made a fix for this issue in version [1.0.4](https://github.com/MohammedAlaaMorsi/RangeSeekBar/releases/tag/1.0.4) this happen because I override this parent class method in the library now you can listen by this implementation
```kotlin
rangeSeekBarView.setOnRangeSeekBarViewChangeListener(object : OnRangeSeekBarChangeListener {
            override fun onProgressChanged(seekBar: RangeSeekBarView?, progress: Int, fromUser: Boolean) {
                Log.e("value",progress.toString())
            }

            override fun onStartTrackingTouch(seekBar: RangeSeekBarView?) {

            }

            override fun onStopTrackingTouch(seekBar: RangeSeekBarView?) {

            }

        })
```
*noted that if you `setAnimated` to `true` you will get only the last value

**Comment by lobothijau** [comment]
Works like a charm. Thank you very much.
